src/plugins/plot/axis/YAxis.vue

```diff
@@ -113,18 +119,20 @@ export default {
             return this.singleSeries === true || this.hasSameRangeValue === true;
         },
         yAxisStyle() {
-            const isMainYAxis = this?.id === this.mainYAxisId;
             let style;
             const width = `width: ${this.tickWidth + 20}px`;
-            const border = `border-right: 1px solid`;
-            if (isMainYAxis) {
-                if (this.visibleYAxes > 1) {
-                    style = `${width}; left: ${this.plotWidth - this.tickWidth + 40}px`;
+            const multipleTickOffset = this.multipleLeftAxes ? 20 : 0;
+            const border = this.multipleLeftAxes ? `border-right: 1px solid` : '';
+
+            if (this.position === 'left') {
+                const offset = (this.id - 1);
+                if (offset) {
+                    style = `${width}; ${border}; left: ${this.plotLeftTickWidth - this.tickWidth - offset * (multipleTickOffset + 5)}px`;
                 } else {
-                    style = `${width}; left: ${this.plotWidth - this.tickWidth}px`;
+                    style = `${width}; left: ${this.plotLeftTickWidth - this.tickWidth + multipleTickOffset}px`;
                 }
-            } else {
-                style = `${width}; ${border}; left: ${this.plotWidth - this.tickWidth - 3}px`;
+            } else if (this.position === 'right') {
```

Comment:
can this just be an `else` if the position has a default value of 'left' ?

Reply:
Done